scripts/jenkins-common.sh

```diff
@@ -24,20 +24,20 @@ fi
 
 # Reset the jenkins worker's virtualenv back to the
 # state it was in when the instance was spun up.
-if [ -e $HOME/edx-venv_clean.tar.gz ]; then
-    rm -rf $HOME/edx-venv
-    tar -C $HOME -xf $HOME/edx-venv_clean.tar.gz
-fi
+# if [ -e $HOME/edx-venv_clean.tar.gz ]; then
+#     rm -rf $HOME/edx-venv
+#     tar -C $HOME -xf $HOME/edx-venv_clean.tar.gz
+# fi
 
```

Comment:
Should we keep commented code

Reply:
Removed after discussion with Muhammad Ali